doc/sphinx/addendum/type-classes.rst

```diff
@@ -347,10 +347,19 @@ Summary of the commands
    matching that pattern will trigger use of the instance.  Otherwise,
    use is triggered based on the conclusion of the type.
 
-   This command supports the :attr:`global` attribute that can be
-   used on instances declared in a section so that their
-   generalization is automatically redeclared when the section is
-   closed.
+   This command supports the following locality attributes.
+
+   + :attr:`local` makes the instance disappear when the enclosing module or
+     section is closed.
+
+   + :attr:`global` instances are automatically made available when the module
+     that defines them is required. In particular, for instances declared in a
+     section their generalization is automatically redeclared as :attr:`global`
+     when the section is closed.
+
+   + :attr:`export` instances are only made available when the module that
+     defines them is imported, similarly to hints. This attribute is not allowed
+     inside sections.
```

Comment:
Probably better not to give full explanations for every command that uses these attributes.  They are repetitive and thus dull to read, also users may spend time pondering the significance of wording variations between different commands (i.e. is there a functional difference of not).

If the main explanation (e.g. [here](https://coq.inria.fr/distrib/current/refman/language/core/modules.html#coq:attr.local)) of any of these 3 is incomplete, it's better to tweak that.

Any behavior specific to Instance should be explained here, of course.

```suggestion
This command supports the :attr:`local`, :attr:`global` and :attr:`export attributes
```

Reply:
It's a bit dangerous to rely on a semantics by reference since it can always change there without anybody realizing, but OK.

Reply:
Achieving consistency in software and its documentaron is not easy.  It often requires eternal vigilance. Théo and I  spend a lot of time on it as we've been updating the doc.